pkgs/development/libraries/thrift/0.10.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, boost, zlib, libevent, openssl, python, pkgconfig, bison
+, flex, twisted
+}:
+
+stdenv.mkDerivation rec {
+  name = "thrift-${version}";
+  version = "0.10.0";
+
+  src = fetchurl {
+    url = "https://archive.apache.org/dist/thrift/${version}/${name}.tar.gz";
+    sha256 = "02x1xw0l669idkn6xww39j60kqxzcbmim4mvpb5h9nz8wqnx1292";
+  };
+
+  #enableParallelBuilding = true; problems on hydra
+
+  # Workaround to make the python wrapper not drop this package:
+  # pythonFull.buildEnv.override { extraLibs = [ thrift ]; }
```

Comment:
That's not how it should be done. Apparently this provides Python bindings. Those bindings should be made available via `python-packages.nix`.

Reply:
Thanks for pointing that out!

Would you be okay if I look at that in a separate PR?

All I did here with `thrift` here is to reactivate the existing `0.10` thrift from `nixpkgs`, it's the vanilla `thrift 0.10` so to speak.